### PR TITLE
Keep players in group channel after using a bettershard portal

### PR DESCRIPTION
**Pull request by idoash4**

### CODE REVIEW

File: src/vg/civcraft/mc/civchat2/listeners/BetterShardsListener.java [hunk 1]
```diff
@@ -26,12 +26,15 @@ public BetterShardsListener(CivChat2 plugin) {
 	public void PlayerChangeServer(PlayerChangeServerEvent event){
 		String playerName = NameAPI.getCurrentName(event.getPlayerUUID());
 		UUID playerToReplyUUID = chatMan.getPlayerReply(playerName);
-		if (playerToReplyUUID == null) {
-			return;
+		if (playerToReplyUUID != null) {
+			String playerToReply = NameAPI.getCurrentName(playerToReplyUUID);
+			if(playerToReply != null){
+				MercuryAPI.sendMessage(event.getServerTravelingTo(), "channel" + sep + "reply" + sep + playerName + sep + playerToReply, "civchat2");
```
Comment (Maxopoly) [inline src/vg/civcraft/mc/civchat2/listeners/BetterShardsListener.java:32]: return here

Comment (idoash4) [inline src/vg/civcraft/mc/civchat2/listeners/BetterShardsListener.java:32]: Why? A player can be in a group channel and have someone to reply to. In this case we want to send 2 messages.

Comment (Maxopoly) [inline src/vg/civcraft/mc/civchat2/listeners/BetterShardsListener.java:32]: Isnt this the thing handling if you are in a chatchannel? The thing when you just run /tell blackxnt for example

Comment (idoash4) [inline src/vg/civcraft/mc/civchat2/listeners/BetterShardsListener.java:32]: No. I actually forgot a player can be in a channel with a player and not only with a group. Currently this change only handles /r and group channels. I will add private channels as well.

**Comment by idoash4** [comment]
This is safe to merge

**Pull request merged**